docs/source/faq/faq.md

```diff
@@ -12,10 +12,11 @@ This is a growing set of technical FAQs. The [product FAQs on the Kedro website]
 * {doc}`Where can I find the documentation about Kedro-Viz<kedro-viz:kedro-viz_visualisation>`?
 * {py:mod}`Where can I find the documentation for Kedro's datasets <kedro-datasets:kedro_datasets>`?
 
-## Working with Jupyter
+## Working with Notebooks
 
 * [How can I debug a Kedro project in a Jupyter notebook](../notebooks_and_ipython/kedro_and_notebooks.md#debugging-a-kedro-project-within-a-notebook)?
 * [How do I connect a Kedro project kernel to other Jupyter clients like JupyterLab](../notebooks_and_ipython/kedro_and_notebooks.md#ipython-jupyterlab-and-other-jupyter-clients)?
+* [How can I use the Kedro IPython extension in a notebook where launching a new kernel is not an option](../notebooks_and_ipython/kedro_and_notebooks.md#exploring-the-project-with-the-kedroipython-extension)?
```

Comment:
https://github.com/kedro-org/kedro/pull/3529/files

I have some initial draft PR previously. I had a different header because I think people mostly copy-and-paste the error and search. This header is descriptive but I also think it's very unlikely people can search this. I don't know what's the preference here, maybe @stichbury has better insight,